Lib/test/test_syntax.py

```diff
@@ -1015,15 +1015,15 @@ def test_assign_del(self):
         self._check_error("del f(), x", "delete function call")
```

Comment:
Honestly all these checks would be clearer to the reader if they included the word "cannot" (which IIUC is part of the error message).